plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/CubridConstants.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2023 DBeaver Corp and others

```

Comment:
```suggestion
 * Copyright (C) 2010-2024 DBeaver Corp and others

```

Reply:
Please update the year in the copyrights.

Reply:
I have updated it.